multiples.py

```diff
@@ -15,6 +16,14 @@ def good(i):
     return by_3 or by_5
 
 
+def even_elements_mult(i):
+    result = 1
+    for num in i:
+        if num % 2 == 0 and num > 0:
```

Comment:
You will be able to remove second condition `num > 0` if you correctly create for loop using range. How?